### PR TITLE
[6.13.z] host edit: test correct page redirection

### PR DESCRIPTION
**Pull request by pnovotny**
Cherrypick of PR: https://github.com/SatelliteQE/robottelo/pull/14844

### Problem Statement

New test for correct page redirection after editing a host.

Component evaluation - SAT-21625
https://bugzilla.redhat.com/2166303

### Related Issues

Requires https://github.com/SatelliteQE/airgun/pull/1362

### CODE REVIEW

**Comment by pnovotny** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_positive_page_redirect_after_update

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6741
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_positive_page_redirect_after_update --external-logging
Test Result : ========== 1 passed, 37 deselected, 48 warnings in 1125.69s (0:18:45) ==========
```

**Review by pondrejk — APPROVED** [review]

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -1714,3 +1714,25 @@ def test_positive_tracer_enable_reload(tracer_install_host, target_sat):
...
         tracer = session.host_new.get_tracer(tracer_install_host.hostname)
         assert tracer['title'] == "No applications to restart"
+
+
+@pytest.mark.tier3
+@pytest.mark.rhel_ver_match('8')
+def test_positive_page_redirect_after_update(target_sat, current_sat_location):
+    """Check that page redirects correctly after editing a host without making any changes.
+    :id: 29c3397e-0010-11ef-bca4-000c2989e153
+    :steps:
```
Comment (pondrejk) [inline tests/foreman/ui/test_host.py:1724]: I suppose the quality checks are complaining about missing spaces between doscstring items, at least that is the difference compared to the original PR

**Comment by pnovotny** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_positive_page_redirect_after_update

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6745
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_positive_page_redirect_after_update --external-logging
Test Result : ========== 1 passed, 37 deselected, 48 warnings in 693.09s (0:11:33) ===========
```

**Pull request merged**